content/en/tracing/profiling/proxy.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Proxy configuration with Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+Profiles collected by the Datadog libraries are sent directly from your running application to Datadog. If your network configuration is restricted for outbound traffic use your proxy to redirect the profiles to Datadog.
+
+If setup, your proxy should point to the following Datadog profiles endpoints:
+
+- If you are on Datadog US site. `https://intake.profile.datadoghq.com`
+- If you are on Datadog EU site. `https://intake.profile.datadoghq.eu`
+
+## Profiling libraries Proxy Setup
+
+Use the options below to configure your profiling libraries to send the collected profile to your proxy:
+
+{{< tabs >}}
+{{% tab "Java" %}}
```

Comment:
@jd @gbbr do the python and go profilers' underlying http libs have standard proxy config mechanisms?

Reply:
Yes it should be easy to set up a proxy in Go. I'd bet it's the same for Python.

Reply:
Does it mean that the env variables are the same for those two languages ?

Reply:
I’m not sure if the question was whether the Go profiler can be configured to use a proxy or if Go supports proxy setups easily in the standard library. I was answering the latter, for the former the answer is no.

Reply:
I not sure to fully follow as my tech depth on go lang is limited.. 
We will have to include an exemple of setup when the go profiler is GA though.

Reply:
Sorry Gus if this was confusing. I meant to say: we don't have support for this in the profiler today, but we can add it later on if needed.

Reply:
ha gotcha :) thanks for the precision

Reply:
We don't have any API nor settings in the profiler to use an HTTP proxy.

To my surprise, Python builtin HTTP client does not honor the standard `HTTP_PROXY` environment variable. That's something we could enhance.

Reply:
From what I can tell with a quick google, go does respect HTTP_PROXY. @gbbr do you want to verify and then we can document that here? Non-blocking.